pkg/kubelet/dockertools/docker_test.go

```diff
@@ -800,3 +802,23 @@ func TestMilliCPUToQuota(t *testing.T) {
 		}
 	}
 }
+
+const letterBytes = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+func randStringBytes(n int) string {
+	b := make([]byte, n)
+	for i := range b {
+		b[i] = letterBytes[rand.Intn(len(letterBytes))]
+	}
+	return string(b)
+}
+
+func TestLogSymLink(t *testing.T) {
+	as := assert.New(t)
+	containerLogsDir := "/foo/bar"
+	podFullName := randStringBytes(128)
```

Comment:
The test is intentionally not using that value. I'd prefer the test to be independent of the implementation logic.